examples/vue3-ssr-vite/src/pages/Home.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div>
+    <header class="header">
+      <h1 class="header-title">
+        <a href="/">
+          vue-instantsearch-app
+        </a>
+      </h1>
+      <p class="header-subtitle">
+        using
+        <a href="https://github.com/algolia/vue-instantsearch">
+          Vue InstantSearch
+        </a>
+      </p>
+    </header>
+
+    <div class="container">
+      <ais-instant-search-ssr>
+        <div class="search-panel">
+          <div class="search-panel__filters">
+            <ais-panel>
+              <ais-refinement-list attribute="brand" />
+            </ais-panel>
+          </div>
+
+          <div class="search-panel__results">
+            <div class="searchbox">
+              <ais-search-box placeholder="" />
+            </div>
+            <ais-hits>
+              <template v-slot:item="{ item }">
+                <article>
+                  <h1>
+                    <ais-highlight
+                      :hit="item"
+                      attribute="name"
+                    />
+                  </h1>
+                  <p>
+                    <ais-highlight
+                      :hit="item"
+                      attribute="description"
+                    />
+                  </p>
+                </article>
+              </template>
+            </ais-hits>
+
+            <div class="pagination">
+              <ais-pagination />
+            </div>
+          </div>
+        </div>
+      </ais-instant-search-ssr>
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  AisInstantSearchSsr,
+  AisPanel,
+  AisRefinementList,
+  AisSearchBox,
+  AisHits,
+  AisHighlight,
+  AisPagination,
+} from 'vue-instantsearch/dist/vue3/es';
+
+export default {
+  components: {
```

Comment:
`app.use(InstantSearch)` didn't work and gave me this on server side:

```
[Vue warn]: Failed to resolve component: ais-refinement-list
[Vue warn]: Failed to resolve component: ais-panel
[Vue warn]: Failed to resolve component: ais-search-box
[Vue warn]: Failed to resolve component: ais-highlight
[Vue warn]: Failed to resolve component: ais-hits
[Vue warn]: Failed to resolve component: ais-pagination
[Vue warn]: Failed to resolve component: ais-instant-search-ssr
```

Not sure why.

Reply:
because the components aren't copied over in createApp?

Reply:
I just checked and when I plug InstantSearch into the app, the server bundle includes the whole components and still somehow doesn't work.

<details>
<summary>error log
</summary>
<pre>
[Vue warn]: Failed to resolve component: ais-instant-search-ssr
[Vue warn]: Failed to resolve component: ais-panel
[Vue warn]: Failed to resolve component: ais-refinement-list
[Vue warn]: Failed to resolve component: ais-search-box
[Vue warn]: Failed to resolve component: ais-hits
[Vue warn]: Failed to resolve component: ais-highlight
[Vue warn]: Failed to resolve component: ais-pagination
[Vue warn]: Unhandled error during execution of serverPrefetch hook 
  at <Anonymous>
(node:43082) UnhandledPromiseRejectionWarning: TypeError: Invalid value used as weak map key
    at WeakMap.set (<anonymous>)
    at normalizePropsOptions (/Users/eunjaelee/workspace/vue-instantsearch/examples/vue3-ssr-vite/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:3580:15)
    at createComponentInstance (/Users/eunjaelee/workspace/vue-instantsearch/examples/vue3-ssr-vite/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:6973:23)
    at renderComponentVNode (/Users/eunjaelee/workspace/vue-instantsearch/examples/vue3-ssr-vite/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:184:22)
    at Proxy.ssrRenderComponent (/Users/eunjaelee/workspace/vue-instantsearch/examples/vue3-ssr-vite/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:520:12)
    at eval (/src/pages/Home.vue:24:31)
    at renderComponentSubTree (/Users/eunjaelee/workspace/vue-instantsearch/examples/vue3-ssr-vite/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:250:13)
...
</pre>
</details>

Anyway, it's not a good practice to include all the components that might not be used in the server bundle, so I'll keep this as-is.